src/components/tabs/tabs.js

```diff
@@ -0,0 +1,275 @@
+import '../../vendor/polyfills/Function/prototype/bind'
+import '../../vendor/polyfills/Element/prototype/classList'
+import '../../vendor/polyfills/Event' // addEventListener and event.target normaliziation
+import { nodeListForEach } from '../../common'
+
+function Tabs ($module) {
+  this.$module = $module
+  this.$tabs = $module.querySelectorAll('.govuk-tabs__tab')
+
+  this.keys = { left: 37, right: 39, up: 38, down: 40 }
+  this.jsHiddenClass = 'js-hidden'
+}
+
+Tabs.prototype.init = function () {
+  if (typeof window.matchMedia === 'function') {
+    this.setupResponsiveChecks()
+  } else {
+    this.setup()
```

Comment:
Previously, this feature detect was around the whole component which meant that on small screens no tabs will be setup.

Am I right in thinking that in browsers that don't support matchMedia, they'll get tabs regardless of screen size?

Reply:
Given that the mobile browsers that we support all have [good support for matchMedia](https://caniuse.com/#search=matchmedia) maybe this isn't too much of a worry.

With the assumption that most of the time IE9 and below are used by users on desktop machines with wide screens?

Reply:
Yes, it's probably okay. Just to note two examples where it's not as robust:

An IE9 user who opens up two windows side by side will get tabs in a small-ish screen.

Similarly, Blackberry 7, for example, will get tabs on their small screen.

In both cases, they will experience some layout issues such as horizontal scrolling or wrapping of tabs.